src/app/core/messages/view-all-messages/view-all-messages.component.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../../../styles/overrides';
+@import '../../../../styles/shared';
+
+$card-width: 800px;
+$card-icon-width: 31px;
+
+.card-message {
+	border: none;
+	-webkit-box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.298039215686275);
+	box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.298039215686275);
+	display: inline-block;
+	width: 100%;
+	max-width: $card-width;
```

Comment:
should we add min-width too?

Reply:
I just added a min-width above the cards on `<section>`.  This ensure padding is maintained.